python/ray/cloudpickle/cloudpickle_fast.py

```diff
@@ -24,24 +24,44 @@
 
 from enum import Enum
```

Comment:
You probably want to leave this file untouched since it is copied from upstream, right?